scripts/teamshow

```diff
@@ -46,10 +46,8 @@ class Teamshow(object):
         """
             Get the portchannel names from minigraph.
         """
```

Comment:
Can you update the comment of "Get the portchannel names from minigraph" accordingly?

Reply:
hi, please review again, thanks